apps/parsley/src/pages/LogView/LoadingPage/useResolveLogURLAndRenderingType.ts

```diff
@@ -1,11 +1,13 @@
 import { useQuery } from "@apollo/client";
 import { LogRenderingTypes, LogTypes } from "constants/enums";
 import {
-  getJobLogsURL,
+  getEVGJobLogsURL,
```

Comment:
nit. I still prefer `getEvergreenJobLogsURL` since we don't use the EVG abbreviation for function names in this repo